content/en/docs/collector/deployment-methods.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Deployment Methods
```

Comment:
Consider:

- Changing the title to be just **Deployment**
- And rename the file toe `deployment.md`

@cartermp WDYT?

Reply:
fixed at 0941baf